blog/_posts/2016-07-15-team-joda-says-hello.md

```diff
@@ -0,0 +1,49 @@
+
+The “da” in joda stands for Dayana:
+“I studied literature in Colombia, worked as a teacher for a couple of years and then came to Europe for a Masters Degree in Media and Culture. I had consistently tried to learn programming and had consistently failed until I went to a Rails Girls Berlin Workshop where jo + da met. I am hoping to learn as much as I can to be able to combine my interests in sounds and arts in general with programming and I am really 😃
+to have this opportunity. “
+
+“jo” and “da”  decided to dive into code during this summer and learn intensively together taking part in the Rails Girls Summer of Code program as joda, after working in both project groups together. They now have a great team of coaches, mentors and an awesome supervisor, as well as Yoda’s teachings as a guide:
+
+## “Do or do not. There is no try.”
+We are working on SoundDrop, a project that has participated twice on previous Rails Girls Summer of Code editions and towards which we felt keen on from the beginning: The app lets you record sounds and connect them to a place where they belong, thus allowing users to leave an audible mark - anywhere. At the moment, SoundDrop is only a web app, but it needs to go mobile so that users can find drops wherever they are located. So the next milestone for SoundDrop (but of course also for team joda) is to build an API, which we ~~try to~~ do achieve this summer.
+
+## “You will find only what you bring in.”
+In our first week and a half participating in the program, we of course haven’t “brought in” major features yet. We tried to get accustomed to the CSS/front end parts of the app by refactoring some of the code and fixing misplaced buttons and containers. Soon, you will be able to listen to SoundDrops (as the play button works now, yay!) and enjoy the responsive design even on your tablet screen :)
+Currently we are preparing for the great task of building a json API for SoundDrop. For this goal we built our own small API and now practice testing on it. Once we’ve tested the whole CRUD cycle, we want to use our knowledge for the “big” SoundDrop API.
+Furthermore we profit a lot from the many deep dive sessions by our coaches, that provide us with theoretical “background” input.
+
+![Team joda](img/blog/2016/team_joda_collage.jpg)<font color="grey"><small><i>Team joda, coaches and one mentor (Image: Team joda)</i></small></font>
+
+
+## “Always pass on what you have learned.”
+
+Almost two weeks into the program, we have already discovered some best practices we would like to share with our readers:
+* Take breaks regularly (e.g. 5 minutes every hour)
```

Comment:
for this to show as a bullet list correctly, you need to add an empty line between `readers:` and the first list item :)